Dockerfile

```diff
@@ -36,6 +38,8 @@ RUN sbt compile
 COPY src/main src/main
 COPY src/test src/test
 RUN sbt compile
+RUN sbt test
```

Comment:
yeah, we certainly don't have to ... but they're also pretty quick, and will catch any environmental errors (works locally, fails in docker, or whatever) so I thought I'd toss 'em in.